substrate/frame/broker/src/lib.rs

```diff
@@ -550,16 +550,27 @@ pub mod pallet {
 		///
 		/// - `origin`: Must be Root or pass `AdminOrigin`.
 		/// - `initial_price`: The price of Bulk Coretime in the first sale.
-		/// - `core_count`: The number of cores which can be allocated.
+		/// - `total_core_count`: This is the total number of cores the relay chain should have
+		/// after the sale concludes.
+		///
+		/// NOTE: This function does not actually request that new core count from the relay chain.
+		/// You need to make sure to call `request_core_count` afterwards to bring the relay chain
+		/// in sync.
+		///
+		/// When to call the function depends on the new core count. If it is larger than what it
+		/// was before, you can call it immediately or even before `start_sales` as non allocated
+		/// cores will just be `Idle`. If you are actually reducing the number of cores, you should
+		/// call `request_core_count`, right before the next sale, to avoid shutting down tasks too
+		/// early.
 		#[pallet::call_index(4)]
-		#[pallet::weight(T::WeightInfo::start_sales((*core_count).into()))]
+		#[pallet::weight(T::WeightInfo::start_sales((*total_core_count).into()))]
 		pub fn start_sales(
 			origin: OriginFor<T>,
 			initial_price: BalanceOf<T>,
-			core_count: CoreIndex,
+			total_core_count: CoreIndex,
```

Comment:
Based on the comment above the parameter should be renamed:

```suggestion
			concluding_core_count: CoreIndex,
```
or 

```suggestion
			resulting_core_count: CoreIndex,
```